dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -592,15 +609,42 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 										secondaryDateline={
 											article.webPublicationSecondaryDateDisplay
 										}
-										isCommentable={showComments}
+										isCommentable={true} // todo
```

Comment:
What the plan for this?

Reply:
no plan. Should have been reverted! It was just to see the layout with comments in context better.